plugins/guisupport/guisupport.cpp

```diff
@@ -481,9 +481,11 @@ void GuiSupport::registerVariantHandler()
 #if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
 void GuiSupport::updateWindowTitle(QWindow *w)
 {
-    if (w->title().endsWith(m_titleSuffix))
-        return;
-    w->setTitle(w->title() + m_titleSuffix);
+    if (w) {
```

Comment:
don't set a default value for window

Reply:
it doesn't make much sense to call the function without setting a window.
So instead of checking for w != null just don't pass a default parameter.

Reply:
Fixed